test/resource.js

```diff
@@ -169,7 +171,8 @@ tap.test('resource.fetch() - redirectable flag - podlet responds with 302 redire
 
     t.equal(result.content, '');
     t.same(result.headers, {
-        connection: 'close',
+        connection: 'keep-alive',
+        'keep-alive': 'timeout=5',
```

Comment:
There's a change from connection close to connection keep alive with the rewrite to Undici. We agreed that this was acceptable @trygve-lie ?

Reply:
:+1: